Dockerfile

```diff
@@ -1,7 +1,21 @@
-FROM bitnami/node
+FROM bitnami/node:7
+
+MAINTAINER Bitnami <containers@bitnami.com>
+
+ENV NODE_ENV=production
+
+RUN npm install yarn eslint  --global
+
+COPY . /app
+
+WORKDIR /app
+
+RUN yarn install
+
+RUN npm rebuild node-sass
+
+RUN yarn run compile
```

Comment:
Sure. I've split into different steps to be easier to debug, but now it makes sense rejoin all them.